webapp/src/ts/services/startup-modals.service.ts

```diff
@@ -20,19 +15,9 @@ interface StartupModal {
 })
 export class StartupModalsService {
 
-  initialized;
-  tours: object[];
   modalsToShow: StartupModal[];
   startupModals: StartupModal[] = [
-    // welcome screen
-    {
-      required: settings => !settings.setup_complete,
-      render: () => {
-        return this.modalService.show(WelcomeComponent, { class: 'welcome' })
-          .catch(() => {});
-      },
-    },
-    // guided setup
+    // Guided Setup
```

Comment:
I suppose this file has to go after both tour and guided setup are removed, right?

Reply:
Correct!